src/main/java/com/hillel/items_exchange/exception/BadRequestException.java

```diff
@@ -6,6 +6,9 @@
 @ResponseStatus(HttpStatus.BAD_REQUEST)
 public class BadRequestException extends RuntimeException {
 
+    public BadRequestException() {
```

Comment:
Can we use lombok annotation @NoArgsConstructor for this?

Reply:
Got it!